### ISSUE TITLE
Listen for all fields are valid without show errors 

### ISSUE THREAD

**Issue by hienhodeveloper** [flutter-form-builder-ecosystem/flutter_form_builder#794]
I want to disable/enable submit button when all fields are validated correctly. How can I archive it? I try using "onChanged" like this. It works, but I don't want all my fields to show errors for every changed

```dart
FormBuilder(
  key: _formKey,
  onChanged: () {
    var newIsValid = _formKey.currentState?.validate() ?? false;
    if (newIsValid != isValid) {
      setState(() {
        isValid = newIsValid;
      });
    }
  },
)
child: Column()
```

**Comment by YouSour** [comment]
i want to know too , how ?

**Comment by deandreamatias** [comment]
Has a some solutions for this, every of them depends of state management.
You can take a look in autovalidateMode

**Comment by renittobb** [comment]
I think you can use _formKey.currentState?.isValid  instead of validate() method

**Comment by deandreamatias** [comment]
Please for this case, use `isValid`.
Now that I solved this issue #1157, this work like expected
